tyche-update-helper.php

```diff
@@ -34,3 +35,24 @@ function ts_modify_curl_request( &$handle, $args, $url ) {
 }
 
 add_action( 'http_api_curl', 'ts_modify_curl_request', 10, 3 );
+
+/**
+ * Function orddd_remove_transient_from_rest_api.
+ *
+ * @version 2.0.0
+ */
+function orddd_remove_transient_from_rest_api() {
+	$orddd_is_transient_removed = get_option('orddd_is_transient_removed');
+	if ( ! $orddd_is_transient_removed ) {
```

Comment:
There must be an additional check to see the Order Delivery Date plugin is installed else there would be fatal errors.